README.md

```diff
@@ -53,13 +53,13 @@ To delete the generated artifacts, execute below command:
 Supported key sizes are 128, 192 and 256.
 
 	Example:
-	# test-bench.sh run keygen --algorithm AES --keysize 128 --transient=true
+	# test-bench.sh run keygen --algorithm AES --keysize 128 --transient true --threadcount 5 --time 30
```

Comment:
Thanks for updating the README with missed documentation. However the `--file-idf` is still not described in the README.